search-service/src/main/kotlin/com/egm/stellio/search/temporal/util/TemporalQueryParamsUtils.kt

```diff
@@ -1,10 +1 @@
 package com.egm.stellio.search.temporal.util
-
-const val TIMEREL_PARAM = "timerel"
-const val TIMEAT_PARAM = "timeAt"
-const val ENDTIMEAT_PARAM = "endTimeAt"
-const val AGGRPERIODDURATION_PARAM = "aggrPeriodDuration"
-const val AGGRMETHODS_PARAM = "aggrMethods"
-const val LASTN_PARAM = "lastN"
-const val TIMEPROPERTY_PARAM = "timeproperty"
-const val WHOLE_TIME_RANGE_DURATION = "PT0S"
```

Comment:
`WHOLE_TIME_RANGE_DURATION` is not a query param (just a special value we have to take care of when calculating aggregations, more like an implementation detail)

Reply:
Yes i put it in TemporalQueryUtils instead. (because it is only used by search service)